corehq/apps/users/views/__init__.py

```diff
@@ -967,9 +967,9 @@ def post(self, request, *args, **kwargs):
                     domain_request = DomainRequest(**data)
                     domain_request.send_request_email()
                     domain_request.save()
-                    domain = Domain.get_by_name(domain_request.domain)
+                    domain_obj = Domain.get_by_name(domain_request.domain)
                     return render(request, "users/confirmation_sent.html", {
-                        'hr_name': domain.display_name() if domain else domain_request.domain,
+                        'hr_name': domain_obj.display_name() if domain_obj else domain_request.domain_obj,
```

Comment:
The last one on this line should not be changed. `DomainRequest.domain` is a string, and `DomainRequest.domain_obj` is not a thing.

https://github.com/dimagi/commcare-hq/blob/d7301f0bac0bdd33124f294d9067c86e9617bbd5/corehq/apps/users/models.py#L2512

Reply:
thanks - 3e83f6454f2